reference/7/Microsoft.PowerShell.Utility/Out-GridView.md

```diff
@@ -0,0 +1,427 @@
+
+This example shows how to send data collected from remote computers to `Out-GridView`.
+
+```powershell
+Invoke-Command -ComputerName S1, S2, S3 -ScriptBlock {Get-Culture} | Out-GridView
+```
+
+`Invoke-Command` runs `Get-Culture` on three remote computers. The resulting data is piped to
+`Out-GridView`. Notice that the script block that runs on the remote computer does not include the
+`Out-GridView` command. If it did, the command would fail when it tried to open a grid view window
+on each of the remote computers.
+
+### Example 7: Pass multiple items through `Out-GridView`
+
+This example lets you select multiple processes from the `Out-GridView` window. The processes that
+you select are passed to the `Export-Csv` command and written to the `ProcessLog.csv` file.
+
+```powershell
+Get-Process | Out-GridView -PassThru | Export-Csv -Path .\ProcessLog.csv
+```
+
+The **PassThru** parameter of `Out-GridView` lets you send multiple items down the pipeline. The
+**PassThru** parameter is equivalent to using the **Multiple** value of the **OutputMode**
+parameter.
+
+### Example 8: Create a Windows shortcut to `Out-GridView`
+
+This example shows how to use the **Wait** parameter of `Out-GridView` to create a Windows shortcut to the `Out-GridView` window.
+
+```powershell
+pwsh -Command "Get-Service | Out-GridView -Wait"
+```
+
+This command line can be used in a Windows shortcut. Without the **Wait** parameter, PowerShell
+would exit as soon as the `Out-GridView` window opened, which would close the `Out-GridView` window
+almost immediately.
+
+## PARAMETERS
+
+### -InputObject
+
+Specifies object that the cmdlet accepts as input for `Out-GridView`.
+
+When you use the **InputObject** parameter to send a collection of objects to `Out-GridView`,
+`Out-GridView` treats the collection as one collection object, and it displays one row that
+represents the collection. To display the each object in the collection, use a pipeline operator (|)
+to send objects to `Out-GridView`.
+
+```yaml
+Type: PSObject
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -OutputMode
+
+Specifies the items that the interactive window sends down the pipeline as input to other commands.
+By default, this cmdlet does not generate any output. To send items from the interactive window down
+the pipeline, click to select the items and then click OK.
+
+The values of this parameter determine how many items you can send down the pipeline.
+
+- None.  No items. This is the default value.
+- Single. Zero items or one item. Use this value when the next command can take only one input
+  object.
+- Multiple. Zero, one, or many items. Use this value when the next command can take multiple input
+  objects. This value is equivalent to the **Passthru** parameter.
+
+This parameter was introduced in Windows PowerShell 3.0.
```

Comment:
Does this make sense to keep considering the entire cmdlet is being brought back in 7?

Reply:
Fixed